extensions/kubeapi/rbac/create.go

```diff
@@ -135,3 +172,40 @@ func CreateProjectRoleTemplateBinding(client *rancher.Client, prtb *v3.ProjectRo
 
 	return newprtb, nil
 }
+
+// CreateClusterRoleTemplateBinding is a helper function that uses the dynamic client to create a cluster role template binding for a specific user
+// in the given downstream cluster.
+func CreateClusterRoleTemplateBinding(client *rancher.Client, namespace, crtbName, userName, roleTemplateName, principalID string, annotations, labels map[string]string) (*v3.ClusterRoleTemplateBinding, error) {
```

Comment:
Can we pass the crtb as input to this function instead of having so many input parameters, similar to what we've done for global role [here](https://github.com/rancher/shepherd/blob/80035045f223836a0b072348c378c7a58be885f4/extensions/kubeapi/rbac/create.go#L74)? This approach would make it easier to integrate any new fields introduced in the crtb without having to modify this function.